flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala

```diff
@@ -999,6 +999,47 @@ class Table(
     new OverWindowedTable(this, overWindows.toArray)
   }
 
+  /**
+    * Performs a global aggregate operation with an aggregate function. Use this before a selection
+    * to perform the selection operation. The output will be flattened if the output type is a
+    * composite type.
+    *
+    * Example:
+    *
+    * {{{
+    *   val aggFunc: AggregateFunction[_, _] = new MyAggregateFunction
+    *   tableEnv.registerFunction("aggFunc", aggFunc);
+    *   table.aggregate("aggFunc(a, b) as (f0, f1, f2)").select('f0, 'f1)
+    * }}}
+    */
+  def aggregate(aggregateFunction: Expression): AggregatedTable = {
+    unwrap(aggregateFunction, tableEnv) match {
```

Comment:
Line 1016:1021 is duplicated with line 1139:1144. How about change Line 1016:1021 to:
`groupBy().aggregate(aggregateFunction)`

Reply:
Done